.travis.yml

```diff
@@ -2,3 +2,19 @@ language: ruby
 cache: bundler
 rvm:
   - 2.5.1
+
+dist: trusty
+sudo: required
+addons:
+  chrome: stable
+
+before_install:
+  - wget https://chromedriver.storage.googleapis.com/2.35/chromedriver_linux64.zip
+  - unzip chromedriver_linux64.zip -d spec/
+
+before_script:
+  - "export PATH=$PATH:`pwd`/spec/"
+
+script:
```

Comment:
I don't think you need this. By default it runs `bundle exec rake` which I already configured to run rubocop and rspec in `Rakefile`